src/plugins/data/public/autocomplete/collectors/create_usage_collector.ts

```diff
@@ -7,8 +7,9 @@
  */
 
 import { first } from 'rxjs/operators';
-import { StartServicesAccessor } from '../../../../../core/public';
-import { METRIC_TYPE, UsageCollectionSetup } from '../../../../usage_collection/public';
+import { METRIC_TYPE } from '@kbn/analytics';
```

Comment:
Normalizing this import so we don't need to re-export it from the `usageCollection` plugin